nearlib/wallet-account.js

```diff
@@ -77,8 +77,9 @@ class WalletAccount {
     }
 
     receiveMessage(event) {
-        if (event.origin != this._walletBaseUrl) {
+        if (event.source.location.href != this._walletBaseUrl + '/embed/') {
```

Comment:
I am not sure I understand motivation for the change. MDN recommends verifying `origin` and possibly `source`:
https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage

Reply:
for whatever reason.. `origin` did not include the port, and therefore if `walletBaseUrl` includes a port, this will fail.

Reply:
we prob want to check both `origin` (only domain) and `source` (full match) then

Reply:
source gives you a window.. how this is the way to get the full match. i don't see the reason to check origin also.

Reply:
Mozilla for some reason suggests to check it. Maybe window can spoof it's location?

Reply:
fixed